src/screens/SUE/SueReportScreen.tsx

```diff
@@ -235,6 +235,9 @@ export const SueReportScreen = ({
 
   /* eslint-disable complexity */
   const alertTextGeneratorForMissingData = () => {
+    const requiredInputs = data?.[currentProgress]?.requiredInputs;
+    const isAnyInputMissing = requiredInputs?.some((inputKey) => !getValues()[inputKey]);
```

Comment:
why was `: keyof TValues` removed?

Reply:
I didn't copy and paste the code directly, I tried different ways of doing it. It must have been deleted in the meantime. I will add it again.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/db01509d0409ae1312a5a0d859327d8d7cf3ca49